cmd/kube-apiserver/app/options/validation.go

```diff
@@ -130,6 +130,7 @@ func (s CompletedOptions) Validate() []error {
 	var errs []error
 
 	errs = append(errs, s.CompletedOptions.Validate()...)
+	errs = append(errs, s.CloudProvider.Validate()...)
```

Comment:
@liggitt or @andrewsykim why I vaguely remember to hear or read that the kube-apiserver is not really affected by the cloud provider option?

Reply:
https://github.com/kubernetes/kubernetes/pull/120903#issuecomment-1738243511

only the single admission plugin needs it

Reply:
Which means?

Reply:
> Which means?

nothing to this PR, I don't think